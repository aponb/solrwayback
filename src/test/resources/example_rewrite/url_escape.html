<html lang="en">
<head>
    <title>Some page with tricky URL</title>
</head>
<body>
<!-- Ampersand should be technically encoded but this is routinely not the case -->
<p><a href="http://example.com/?foo=bar&zoo=boom&track=o1">Ampersand</a></p>
</body>
</html>
