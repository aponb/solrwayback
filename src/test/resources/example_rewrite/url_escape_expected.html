<html lang="en">
<head>
<title>Some page with tricky URL</title>
</head>
<body>
<!-- Ampersand should be technically encoded but this is routinely not the case -->
<p><a href="http://localhost:0000/solrwayback/services/web/20200430130700/http://example.com/?foo=bar&amp;zoo=boom&amp;track=o1">Ampersand</a></p>
</body>
</html>